src/app/core/services/usage/usage.service.ts

```diff
@@ -1,18 +1,18 @@
 import { FirebaseAnalyticsService } from './firebase-analytics.service'
 import { Injectable } from '@angular/core'
 import { KafkaService } from '../kafka/kafka.service'
+import { LogService } from '../misc/log.service'
 import { SchemaType } from '../../../shared/models/kafka'
 import { UsageEventType } from '../../../shared/enums/events'
 import { WebIntent } from '@ionic-native/web-intent/ngx'
-import { LogService } from '../misc/log.service'
 
 @Injectable()
 export class UsageService {
   constructor(
     private webIntent: WebIntent,
     private kafka: KafkaService,
     private firebaseAnalytics: FirebaseAnalyticsService,
-    private logger: LogService,
```

Comment:
The last comma was intentional, to avoid a git line change if another service gets added at a later time. It's also fine to remove it I suppose.

Reply:
Sorry about that, my formatter automatically removed it. Is it fine if we keep it that way?

Reply:
Sure, no problem.